htslib/vcf.h

```diff
@@ -142,12 +154,14 @@ extern uint8_t bcf_type_shift[];
 #define VCF_SNP     (1<<0)
 #define VCF_MNP     (1<<1)
 #define VCF_INDEL   (1<<2)
-#define VCF_OTHER   (1<<3)
-#define VCF_BND     (1<<4)      // breakend
-#define VCF_OVERLAP (1<<5)      // overlapping deletion, ALT=*
-#define VCF_INS     (1<<6)      // implies VCF_INDEL
-#define VCF_DEL     (1<<7)      // implies VCF_INDEL
+#define VCF_OTHER   32
```

Comment:
Not sure why we need to be different from the origin vcf.h here. @mlathara any ideas?

Reply:
Seems to work just fine in default values. Passes all tests when it is like following, all default values except the ones we added: 
```
#define VCF_REF         0
#define VCF_SNP     (1<<0)
#define VCF_MNP     (1<<1)
#define VCF_INDEL   (1<<2)
#define VCF_OTHER   (1<<3)
#define VCF_BND     (1<<4)      // breakend
#define VCF_OVERLAP (1<<5)     // overlapping deletion, ALT=*
#define VCF_INS     (1<<6)      // implies VCF_INDEL
#define VCF_DEL     (1<<7)      // implies VCF_INDEL
#define VCF_ANY     (VCF_SNP|VCF_MNP|VCF_INDEL|VCF_OTHER|VCF_BND|VCF_OVERLAP|VCF_INS|VCF_DEL)       // any variant type (but not VCF_REF)
#define VCF_NON_REF (1<<8)
#define VCF_SPANNING_DELETION (1<<5)
```